### ISSUE TITLE
Error when I clear cache (discriminator map)

### ISSUE THREAD

**Issue by rosie607** [concrete5-community-store/community_store#422]
Does anyone know what's going on here.
When I clear the cache I get the following error on the store page, orders page and order detail page.

`Entity 'Concrete\Package\CommunityStore\Entity\Attribute\Key\StoreOrderKey' has to be part of the discriminator map of 'Concrete\Core\Entity\Attribute\Key\Key' to be properly mapped in the inheritance hierarchy. Alternatively you can make 'Concrete\Package\CommunityStore\Entity\Attribute\Key\StoreOrderKey' an abstract class to avoid this exception from occurring.`

**Comment by Mesuva** [comment]
Can you give a bit more context to this, did you try to do any updates, or is it just a fresh install of 2.0.5?  Is the error still persisting?

What happens when you hit the Refresh Entities button at /index.php/dashboard/system/environment/entities?

Anything else you can think of that might be relevant is appreciated!

**Comment by Mesuva** [comment]
Did you end up resolving this?

**Comment by rosie607** [comment]
I can't say I resolved it. The issue happened every-time I cleared the cache. I tried deleting all the orders, uninstalling the store, deleting all the users blar blar and it kept happening, I reloaded the database from the previous point before clearing the cache and tried various things and it kept happening. I was at a point in the project where I couldn't stop to look into it in anymore detail so I continued with the site but didn't clear the cache. Sometime later... when I did try to clear the cache again it was fine... So I don't know what to say...
